src/chrome/content/rules/Weibo.com.xml

```diff
@@ -1,22 +1,21 @@
 <!--
-	Other Weibo coverage:	
+	Other Weibo coverage:
 		Weibo.cn.xml
 		Weibo.com-mixedcontent.xml
 		WeiboPay.com.xml
 
 	Insecure cookies are set for these domains and hosts:
-		- .weibo.com
-		- game.weibo.com
-		- sg2.game.weibo.com
-		- yahoo.tw.weibo.com
-
-	Different http/https content:
-		s.weibo.com
+		.weibo.com
+		game.weibo.com
+		sg2.game.weibo.com
+		yahoo.tw.weibo.com
 
 	MCB:
-		chart.weibo.com
-		e.weibo.com
+		chart.weibo.com		( Too many redirects )
+		e.weibo.com			( https://e.weibo.com/register/register )
```

Comment:
mixedcontent pages' contents should be https all.
weibo.com/signup/ redirects to http, no https